python-avd/pyavd/_eos_designs/shared_utils/link_tracking_groups.py

```diff
@@ -21,18 +22,17 @@ class LinkTrackingGroupsMixin(Protocol):
     """
 
     @cached_property
-    def link_tracking_groups(self: SharedUtilsProtocol) -> list | None:
+    def link_tracking_groups(self: SharedUtilsProtocol) -> EosCliConfigGen.LinkTrackingGroups | None:
         if self.node_config.link_tracking.enabled:
```

Comment:
Reverse the logic - needs indentation fix below.
```suggestion
        if not self.node_config.link_tracking.enabled:
            return None
```

Reply:
Done.